core/src/scss/components/nav-toggle/_nav-toggle.scss

```
@@ -0,0 +1,67 @@
+@charset 'UTF-8';
+//
+// Navigation Toggle
+//
+// A clickable drop button as an alternate to an accordian style menu.
+//
+// .su-nav-toggle--light - A lighter option.
+//
+// Markup: ../templates/components/nav-toggle/nav-toggle.twig
+//
+// Style guide: Components.Nav Toggle
+//
+
+.su-nav-toggle {
+  @include margin(0);
+  @include padding(15px 28px);
+  vertical-align: middle;
+  text-align: center;
+  font-size: 0;
+  text-indent: -9999px;
+  overflow: hidden;
+  width: 49px;
+  max-width: 49px;
+  height: 58px;
```

```suggestion
  height: 61px;
```
Center toggle button vertically for 1st level items